src/app/map/leaflet-map/leaflet-map.component.ts

```diff
@@ -0,0 +1,239 @@
+import { Component, AfterViewInit, OnChanges, OnDestroy, Input, OnInit, HostListener, ApplicationRef, Injector, ComponentFactoryResolver } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { LeafletMapUtils } from './leaflet-map.utils';
+import { Project } from 'app/models/project';
+import { ProjectService } from 'app/services/project.service';
+import { ProjectPopupComponent } from 'app/map/leaflet-map/project-popup/project-popup.component';
+
+import 'leaflet';
```

Comment:
Does this import need to happen? Isn't this captured on line 10?

Reply:
Nope. Removed.